src/wrapper.spec.ts

```diff
@@ -308,156 +40,133 @@ describe("writeHandlers", () => {
           },
         },
       ],
-      TracingMode.XRAY,
+      true,
     );
     expect(service).toEqual({
       package: {
-        include: [`${datadogDirectory}/my-lambda.py`, `${datadogDirectory}/**`],
+        include: [`${pythonHandlerFile}`],
       },
     });
   });
 
-  it("adds datadog_handlers to include list by default", async () => {
+  it("redirects js handlers correctly when addLayers is true", async () => {
     mock({});
-    const service = {
-      package: {
-        include: [],
-      },
-    } as any;
-    await writeHandlers(service, [], TracingMode.XRAY);
-    expect(service).toEqual({
-      package: {
-        include: [`${datadogDirectory}/**`],
-      },
-    });
-  });
-
-  it("cleans up existing datadogDirectory", async () => {
-    mock({
-      [datadogDirectory]: {
-        "unused-file": "some-value",
-      },
-    });
     const service = {} as any;
-    await writeHandlers(service, [], TracingMode.XRAY);
-    expect(fs.existsSync(`${datadogDirectory}/unused-file`)).toBeFalsy();
-  });
-
-  it("ignores handlers with poorly formatted names", async () => {
```

Comment:
Wasn't sure if we still want to ignore certain handlers, or just redirect all by default?